# /** ------自定义指令 ------ */

string   au1Type
string   au1Command
uint32   u4ObjectId